test/logformats_test.py

```diff
@@ -374,11 +377,39 @@ def _setup_instance(self):
             adds_default_channel=0,
         )
 
-    def _read_log_file(self, filename):
+    def _read_log_file(self, filename, **kwargs):
         logfile = os.path.join(os.path.dirname(__file__), "data", filename)
-        with can.ASCReader(logfile) as reader:
+        with can.ASCReader(logfile, **kwargs) as reader:
             return list(reader)
 
+    def test_absolute_time(self):
```

Comment:
@felixdivo Added just one test for absolute timestamp since relative is tested by the existing tests

Reply:
Cool! That should suffice.